### DISCUSSION TITLE
Project Beta: ORG wide labels

### DISCUSSION THREAD

**Discussion by edwdev** [community/community#5417] [Projects and Issues]
I have a general project, that has all issues from all repo's linked to it. It would be handy, to be able to create labels - such as "epic". That all repo's can consume, instead of it just being per repo - where we need to create the same "epic" label multiple times.

**Comment by arwidt** [comment]
+1 for this.

**Reply by cpswan** [reply]
Now that there's the .github repo for org wide profiles, issue templates and PR template perhaps that would be the place to define org wide labels.

**Comment by mkvlrn-cm42** [comment]
Isn't this already available from the org settings/repository defaults page?

![image](https://user-images.githubusercontent.com/6718750/137500370-d2b3c98d-f576-4290-926e-a11f973ad28f.png)

It does say "Repository labels" but they will be available for every repo on that org from then on.

Or am I not understanding the request correctly?

**Reply by shiipou** [reply]
> 

It's not really org level tags, it's more a default tag template for new repository. If you add a tag at organization level, it won't be added in your repositories.
To make this I personally set a repo like this : https://github.com/shiipou/labelset
And with a little CI you can easily set your labels in all your repositories.

**Reply by cpswan** [reply]
I have something similar that uses Actions to copy a set of labels across a list of target repos https://github.com/atsign-company/labels
